api/testworkflows/v1/testworkflow_types.go

```diff
@@ -70,8 +70,9 @@ type TestWorkflow struct {
 	Description string `json:"description,omitempty"`
 
 	// TestWorkflow specification
-	Spec   TestWorkflowSpec          `json:"spec" expr:"include"`
-	Status TestWorkflowStatusSummary `json:"status,omitempty"`
+	Spec   TestWorkflowSpec             `json:"spec" expr:"include"`
+	Status TestWorkflowStatusSummary    `json:"status,omitempty"`
+	Health *TestWorkflowExecutionHealth `json:"health,omitempty"`
```

Comment:
I am wondering should this go into the `TestWorkflowStatusSummary` struct.

Reply:
Great suggestion, I'll make the change